src/main/java/com/soyewale/codefellowship/controllers/ApplicationUserController.java

```diff
@@ -18,10 +18,10 @@
     @Autowired private PasswordEncoder passwordEncoder;
 
     @PostMapping("/signup")
-    public RedirectView createNewApplicationUser(String username, String password, String firstName, String lastname, int dateOfBirth, String bio){
+    public RedirectView createNewApplicationUser(String username, String password, String firstName, String lastName, int dateOfBirth, String bio){
```

Comment:
This `int dateOfBirth` caused it to crash when I typed "5/4/82". You can change this to String and then change the 'type' attribute to "date" in the form

Reply:
Thanks. Will make changes accordingly